README.md

```diff
@@ -1,3 +1,5 @@
+<img src="https://electronjs.org/app-img/open-ias-scoreboard/open-ias-scoreboard-icon-128.png">
+
```

Comment:
Could this be put in line with the title? I think it's possible to put it before `Open IAS Scoreboard` to put it inline. This would require setting the height. Maybe set it to something between 1-2 `em`?

Reply:
Hi @gary-kim I will try to add it and  add modification in new PR

Reply:
Alright, should I merge this for now or wait for your new PR?

Reply:
Hi, @gary-kim merge the is PR for now.